### PR TITLE
fix: DEV-3201: Fix naive metric for htmllabels\hypertextlabels

### PR DESCRIPTION
**Pull request by KonstantinKorotaev**

### CODE REVIEW

**Review by guilhermemachado26 — APPROVED** [review]

File: evalme/classification.py [hunk 1]
```diff
@@ -134,7 +134,8 @@ def naive(x, y, per_label=False, **kwargs):
                     t = x[i]['type']
                     # trying to extract label from annotation
                     try:
-                        label = x[i]['value'][t]
+                        label = x[i]['value'][t] if t not in ['htmllabels', 'hypertextlabels'] \
+                            else x[i]['value'].get('hypertextlabels') or x[i]['value'].get('htmllabels')
```
Comment (niklub) [inline evalme/classification.py:138]: rewrite it with
```
label = x[i]['value'].get(t)
if label is None:
 ....
elif isinstance(...)
```

File: evalme/classification.py [hunk 2]
```diff
@@ -143,7 +144,7 @@ def naive(x, y, per_label=False, **kwargs):
                     # No-label if exception occurs
                     except Exception as e:
                         logger.error("Can't assign result for label.", exc_info=True)
-                        label = 'No-label'
+                        continue
```
Comment (niklub) [inline evalme/classification.py:147]: this error is misleading - should be information provided, or there is no error here, hence no exception needed

**Review by niklub — CHANGES_REQUESTED** [review]

File: evalme/classification.py (hunk 1 above)
Comment (KonstantinKorotaev) [reply evalme/classification.py]: Fixed

File: evalme/classification.py (hunk 2 above)
Comment (KonstantinKorotaev) [reply evalme/classification.py:142]: Added type info in error text

Comment (niklub) [reply evalme/classification.py:150]: I can't understand how the exception can happen due to an unexistent `t`,
and why we can't rely on simply retrieving `t` and checking for `labels is None` - without explicit `.get('hypertextlabels')` ? It's very confusing

Comment (KonstantinKorotaev) [reply evalme/classification.py]: > I can't understand how the exception can happen due to an unexistent t

Deleted exception handling on label extraction

> and why we can't rely on simply retrieving t and checking for labels is None - without explicit .get('hypertextlabels') ? It's very confusing

html labels results can contain both hypertextlabels and htmllabels results, so we need to extract either of them

**Review by niklub — APPROVED** [review]

**Pull request merged**